src/parameterized_tendencies/les_sgs_models/smagorinsky_lilly.jl

```diff
@@ -9,7 +9,7 @@ import ClimaCore: Geometry
 """
     lilly_stratification_correction(p, ᶜS)
 
-Return a lazy representation of the Lilly stratification correction factor 
+Return a lazy representation of the Lilly stratification correction factor
```

Comment:
these unrelated changes are a bit distracting. Is there any way you could disable them?